internal/syncronizedmap/syncronizedmap.go

```diff
@@ -0,0 +1,67 @@
+package syncronizedmap
+
+import (
+	"fmt"
+	"sync"
+)
+
+//go:generate mockgen -source=syncronizedmap.go -package=syncronizedmap -destination=mock_syncronizedmap.go
+
+type SyncronizedKernelOsDtkMapping interface {
```

Comment:
Is `Synchronized` needed here?
```suggestion
type KernelOsDtkMap interface {
```

Reply:
Not really, I can remove it.

Reply:
Updated